stdlib/public/core/CTypes.swift

```diff
@@ -193,9 +193,10 @@ func _memcpy(
   let dest = destination._rawValue
   let src = src._rawValue
   let size = UInt64(size)._value
+  let zero = Int32()
   Builtin.int_memcpy_RawPointer_RawPointer_Int64(
     dest, src, size,
-    /*alignment:*/ Int32()._value,
+    /*alignment:*/ zero._value,
```

Comment:
Oh, type constraints on the parameters to `Builtin.int_memcpy_RawPointer_RawPointer_Int64` don’t handle the deduction?  That’s weird.  It’s not like any `Builtin` is overloaded.